css/civicrmNavigation.css

```diff
@@ -91,6 +91,7 @@ li.menu-separator.active{
   display:block;
   font-weight:normal;
   text-decoration:none;
+  border-bottom-style:none;
```

Comment:
As far as i can tell d8 out of the box does not use menu-separator but it was inheriting it from the a tag.  The actual selector we are adding this to is  #civicrm-menu li a, #root-menu-div li a { github is just confusing things